raft/src/main/java/org/apache/kafka/raft/Batch.java

```diff
@@ -120,6 +120,19 @@ public String toString() {
             ')';
     }
 
+    /**
+     * @return a descriptive short String of the batch.
+     */
+    public String describe() {
+        return "Batch(" +
+                "baseOffset=" + baseOffset +
+                ", epoch=" + epoch +
+                ", appendTimestamp=" + appendTimestamp +
+                ", sizeInBytes=" + sizeInBytes +
+                ", lastOffset=" + lastOffset +
+                ')';
+    }
```

Comment:
Why do we need another method, instead of using the `toString` method?

Reply:
`toString` method prints both record lists. I assumed this might be too verbose, hence adding this one without them

Reply:
I can remove this method, and print the epoch, offsets, ts and size "inline" in the `String.format`

Reply:
> I can remove this method, and print the epoch, offsets, ts and size "inline" in the String.format

Yes, I think this is better.

Reply:
Done